stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -588,28 +588,16 @@ public static BStruct createNewEntity(Context context, BStruct struct) {
      * @param context         ballerina context.
      * @param outboundMessage transport message.
      */
-    public static void setKeepAliveAndCompressionHeaders(Context context, HTTPCarbonMessage outboundMessage) {
+    public static void setCompressionHeaders(Context context, HTTPCarbonMessage outboundMessage) {
```

Comment:
Don't we need to change the doc comment with respect to the change? As this method only sets compression header now.

Reply:
Fixed